opennms-doc/guide-admin/src/asciidoc/text/events/event-configuration.adoc

```diff
@@ -123,9 +138,25 @@ Not all events will have values for all tokens, and some refer specifically to i
     The operator instructions for the event.
 `%mouseovertext%`::
     The mouse over text for the event.
+`%tticketid%`::
+    The trouble ticket id associated with the event, if available.
```

Comment:
```suggestion
    The trouble ticket id associated with the event if available.
```

Reply:
Having the comma is better (but optional), but since most of the other instances of "if available" do not, I removed this to be consistent with the other ones, rather than correcting them. :-)